Vision/README.md

```diff
@@ -11,34 +9,34 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
+
+* We put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
+* This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. 
+* We strongly emphasize accessibility, internationalization, privacy, and security.
+* Our standards are rooted in a strong royalty-free patent policy and open copyright licenses, openly developed with consensus of industry and key stakeholders.
+
+# Mission of the W3C
+The fundamental purpose of the W3C is to provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
 
-We will continue to develop a world-wide web that
-* Works for everyone, striving for diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. We strongly emphasize accessibility, internationalization, privacy, security and diversity.
-* Puts the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-* Is trustworthy, by ensuring security and privacy for users. 
-* Does not favor centralization.
-* Implements a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.
-* Is based on standards that are openly developed with consensus of industry and key stakeholders, standards rooted in a strong royalty-free patent policy and open copyright licenses, focussed on interoperability and collective empowerment, and supported by open test suites.
+As the W3C leads the Web forward, our mission is to recognize and embody fundamental values and principles into the architecture of the web.  We must become more principled in our execution of the vision of the Web.
 
-# The Operating Principles of the Consortium
+# Principles and Values
 
-In our operations and membership, we will:
-*	Provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
+We must
+* Ensure the Web is trustworthy, by ensuring security and privacy for users. 
+* Ensure the Web does not favor centralization.
+* Remain focused on interoperability and collective empowerment, and ensure our work is supported by open test suites.
+* Implement a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.
```

Comment:
I think this bullet does not belong in this document, but we can discuss separately.